packages/astro/package.json

```diff
@@ -179,7 +179,7 @@
     "zod": "^3.22.4"
   },
   "optionalDependencies": {
-    "sharp": "^0.33.1"
+    "sharp": "0.32.6"
```

Comment:
This sucks, because users have to do a manual override to use newer versions, but unfortunately, I couldn't figure it out. The semver checks in Sharp cannot really be relaxed, because it (very understandably so) depends on the newer Node API 9. This is a shame because `0.33.0` is otherwise a very very good upgrade making installation much much less troublesome.

Now that Node 20 is the LTS, it's likely that when we'll release 5.0 a majority of our users will be on it, which should make this much easier. In 5.0 we can update our requirements to be `^18.17.0 || >= 20.3.0` or similar, which should cover everything + be understandable by our users.